### PR TITLE
fix note shortcode, lists

### PR DESCRIPTION
**Pull request by kbhawkey**
- fix note shortcode for use in lists
- add example in style guide
- See issue #13314.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Hello, @jimangel. Have cc'ed you here as you raised the issue. Comments welcome!

**Comment by jimangel** [comment]
Great work @kbhawkey! This looks good to me! What are your thoughts on updating the example to demonstrate adding a note in the middle of a list by adding one more item at the end of the list?

I say that because note snippets seemed to work at the top / bottom always. However, I could see it not being needed to prove out this specific issue. I'll let you decide...

/lgtm

**Comment by kbhawkey** [comment]
@jimangel, I will update the list in the style guide.

**Comment by kbhawkey** [comment]
It looks like the note in the list is picking up the font-size (16px) from the list selector in the CSS.
Suggest adding a selector possibly for <li><blockquote><div> to set the font-size to 14px.

File: content/en/docs/contribute/style/style-guide.md [hunk 1]
```diff
@@ -236,6 +235,36 @@ The output is:
 You can _still_ use Markdown inside these callouts.
 {{< /note >}}
 
+You can use a {{</* note */>}} in a list:
+
+```
+1. Use the note shortcode in a list
+
+1. A second item with an embedded note
```
Comment (DanyC97) [inline content/en/docs/contribute/style/style-guide.md]: was this meant to be 2. and so on ?

Comment (kbhawkey) [reply content/en/docs/contribute/style/style-guide.md]: Hello @DanyC97, my mistake. Should be 1's as this text is demonstrating how to write a list.

**Comment by kbhawkey** [comment]
@lucperkins , if time, would you sanity check the `shortcode` changes? Thanks.

**Comment by tengqm** [comment]
/lgtm
defer to @lucperkins for review and approval.

**Comment by lucperkins** [comment]
I don’t have the authority to approve

**Pull request merged**